singular.php

```diff
@@ -21,7 +21,7 @@
 		while ( have_posts() ) {
 			the_post();
 
-			if ( is_page_template( array( 'template-cover.php' ) ) ) {
+			if ( is_page_template( array( 'templates/template-cover.php' ) ) ) {
 				get_template_part( 'template-parts/content-cover' );
```

Comment:
The intent was for this to move to the actual template file instead of the template just calling singular. Keep the template-specific stuff in the template as much as possible.

Reply:
I'm afraid, I don't understand what you mean by that. Would you mind explaining it to me?

Reply:
The template-cover file should contain a loop that uses the content-cover template part.
The singular.php file should contain a loop that uses the content template part.
There should not be a check for the page template in either one.

Reply:
I agree with Joy, this solution that I added when the file(s) were first split is not ideal.

Reply:
@joyously & @carolinan I've adjusted the files as suggested by you in https://github.com/WordPress/twentytwenty/pull/251/commits/cc842420774c7851ff9e73cd06391775f0f23e96